sdk/tests/invariant.test.ts

```diff
@@ -55,4 +56,83 @@ describe('invariant', function () {
 
     assert.deepEqual(newFee, { v: 100 })
   })
+
+  it('should add fee tier', async () => {
+    const { api, account } = await init()
+
+    const invariantData = await getDeploymentData('invariant')
+    const invariant = new Invariant(api, Network.Local)
+
+    const initFee = { v: 10 }
+    const invariantDeploy = await invariant.deploy(
+      account,
+      invariantData.abi,
+      invariantData.wasm,
+      initFee
+    )
+    await invariant.load(invariantDeploy.address, invariantData.abi)
+
+    const feeTier = new FeeTier(600, 5)
+    const anotherFeeTier = new FeeTier(300, 10)
+
+    await invariant.addFeeTier(account, feeTier)
+    let feeTierExists = await invariant.feeTierExist(account, feeTier)
+    let anotherFeeTierExists = await invariant.feeTierExist(account, anotherFeeTier)
+    let feeTiers = await invariant.getFeeTiers(account)
+
+    assert.deepEqual(feeTierExists, true)
+    assert.deepEqual(anotherFeeTierExists, false)
+    assert.deepEqual(feeTiers, [feeTier])
+
+    await invariant.addFeeTier(account, anotherFeeTier)
+    feeTierExists = await invariant.feeTierExist(account, feeTier)
+    anotherFeeTierExists = await invariant.feeTierExist(account, anotherFeeTier)
+    feeTiers = await invariant.getFeeTiers(account)
+
+    assert.deepEqual(feeTierExists, true)
+    assert.deepEqual(anotherFeeTierExists, true)
+    assert.deepEqual(feeTiers, [feeTier, anotherFeeTier])
+  })
+
+  it('should remove fee tier', async () => {
+    const { api, account } = await init()
+
+    const invariantData = await getDeploymentData('invariant')
+    const invariant = new Invariant(api, Network.Local)
+
+    const initFee = { v: 10 }
```

Comment:
Remember to use the literal `n` for all integers. Even if it's redundant, it's worth distinguishing that it is not `number`